src/site/stages/build/process-cms-exports/filters.js

```diff
@@ -57,4 +84,5 @@ function getFilteredEntity(entity) {
 module.exports = {
   getFilteredEntity,
   getFilter,
+  ignoreList,
```

Comment:
Arguably, I don't need to export this anymore. This was just here because it was phase 1 of refactoring.